### PR TITLE
Fix for issue 55

### PR DESCRIPTION
**Pull request by nkoenig**
This prevents `jointVelFeature->SetVelocityCommand(i, 0);` from executing if a `JointForceCmd` component is present.

### CODE REVIEW

**Comment by nkoenig** [comment]
Should be fixed in 9d6f8ba78f04830c8eb9f57941f269a8a618dedc

File: src/systems/physics/Physics.cc [hunk 1]
```diff
@@ -985,7 +987,7 @@ void PhysicsPrivate::UpdatePhysics(EntityComponentManager &_ecm)
             //   JointForceCmd. Remove when it does.
             auto jointVelFeature = entityCast(_entity, jointIt->second,
                 this->entityJointVelocityCommandMap);
-            if (jointVelFeature)
+            if (!force && jointVelFeature)
             {
               jointVelFeature->SetVelocityCommand(i, 0);
```
[the remark this answers is not in the document]
Comment (nkoenig) [reply src/systems/physics/Physics.cc]: Like this: 5c0865262b5309d7e6057d76de48875336d1bddb?

[the remark this answers is not in the document]
Comment (iche033) [reply src/systems/physics/Physics.cc]: @azeey does the velocity command persist on the ign-physics side?I think the problem I ran into before was that if `SetVelocityCommand(i, 0)` is called once on launch, subsequent force commands did not work.

[the remark this answers is not in the document]
Comment (azeey) [reply src/systems/physics/Physics.cc]: hmm, @iche033, I'll have to look into it. The velocity command should only be for one step. It does, however, change the actuator type in DART to an `SERVO`. I'm guessing that calling `SetForce` on the joint once it has been changed to a `SERVO` doesn't work as expected.


@nkoenig 
> Like this: [5c08652](https://github.com/ignitionrobotics/ign-gazebo/commit/5c0865262b5309d7e6057d76de48875336d1bddb)?

Yes!

[the remark this answers is not in the document]
Comment (iche033) [reply src/systems/physics/Physics.cc]: ok I see, thanks for explanation

**Pull request merged**